components/x-gift-article/__tests__/x-gift-article.test.jsx

```diff
@@ -27,70 +29,55 @@ describe('x-gift-article', () => {
 		actions = {}
 
 		fetchMock
-			.get('/article/gift-credits', {
+			.get('path:/article/gift-credits', {
 				allowance: 20,
 				consumedCredits: 5,
 				remainingCredits: 15,
 				renewalDate: '2018-08-01T00:00:00Z'
 			})
-			.get(`/article/shorten-url/${encodeURIComponent(articleUrlRedeemed)}`, {
+			.get(`path:/article/shorten-url/${encodeURIComponent(articleUrlRedeemed)}`, {
 				shortenedUrl: 'https://shortened-gift-url'
 			})
-			.get(`/article/shorten-url/${encodeURIComponent(nonGiftArticleUrl)}`, {
+			.get(`path:/article/shorten-url/${encodeURIComponent(nonGiftArticleUrl)}`, {
```

Comment:
**Question**: Is the `path:` prefix required?

Reply:
I'll leave the `path:` prefix in if thats ok with you.